### DISCUSSION TITLE
dodanie-tunelu-ipv6-do-routera-z-openwrt/

### DISCUSSION THREAD

**Discussion by giscus** [idarek/dariusz.wieckiewicz.org#107] [General]
# dodanie-tunelu-ipv6-do-routera-z-openwrt/

Wolny świat, wolne myśly, wolne słowa.

https://dariusz.wieckiewicz.org/dodanie-tunelu-ipv6-do-routera-z-openwrt/

**Comment by marcinx98x** [comment]
Siemka, wszystko robie zgodnie z przewodnikiem, mam neta w BT ale po ustawieniu WAN6 nie działa mi google.com wywala bład 403, Pomożesz ?

**Reply by idarek** [reply]
> Siemka, wszystko robie zgodnie z przewodnikiem, mam neta w BT ale po ustawieniu WAN6 nie działa mi google.com wywala bład 403, Pomożesz ?

Wygląda na to że twój IP (HE IP) został zbanowany. Spróbuj utworzyć nowy tunel w HE, może dostaniesz z innej puli IP.

Zobacz tutaj: https://support.google.com/websearch/thread/152495336/403-when-connecting-with-ipv6-tunnel-from-he-net

**Reply by marcinx98x** [reply]
Pomogło, miałeś racje, DZIĘKI!!! Twoje poradniki są świetne.

**Reply by idarek** [reply]
Spoiler Alert: właśnie pracuje na czymś innym wiec upewnij się ze śledzisz moja strona przez RSS lub inne dostępne sposoby. Pozdrawiam 😃

**Comment by marcinx98x** [comment]
Dzięki za bardzo szybka odpowiedz :) teraz wiem na czym stoję i będę próbował dalej.

**Comment by spexpl** [comment]
Securebit AG coś nie działa. Tz trafiłem na wyniki z google, które mówią o zmianie adresu strony. Ale szczerze powiedziawczy na "nowa" strona też coś ciężko działa.

**Reply by idarek** [reply]
Wygląda na to że Securebit AG przestało oferować tą "darmową" usługę i przeszła na [model zarobkowy](https://www.securebit.ch/internet/tunnel). Zaktualizowałem (usunąłem) odniesienie do nich z wpisu. Pozdrawiam.
